client/src/components/AuditBoard/Ballot.tsx

```diff
@@ -103,6 +106,47 @@ interface IProps {
   ) => void
 }
 
+interface IRenderedInterpretationProps {
+  interpretation: IBallotInterpretationFormRepresentation
+  contest: IContest
+}
+
+const RenderedInterpretation: React.FC<IRenderedInterpretationProps> = ({
```

Comment:
Nit: `Rendered` is implicit since all components are rendered. Maybe just `Interpretation`?

Reply:
I'd initially named this `Interpretation`, but it conflicted with the imported `Interpretation` enum:

https://github.com/votingworks/arlo/blob/86f7deabc1fda6e561964ab73186aed589b318c9/client/src/components/AuditBoard/Ballot.tsx#L13

I can maybe rename to `InterpretationSummary`